apps/src/sites/studio/pages/lessons/edit.js

```diff
@@ -44,4 +51,24 @@ $(document).ready(function() {
     </Provider>,
     document.getElementById('edit-container')
   );
-});
+}
+
+/**
+ * Initialize the DOM Element and React Component which serve as containers to
+ * display expandable images.
+ *
+ * @see @cdo/apps/src/templates/utils/expandableImages
+ */
+function prepareExpandableImageDialog() {
+  registerReducers({instructionsDialog});
+
+  const container = document.createElement('div');
+  document.body.appendChild(container);
+
+  ReactDOM.render(
+    <Provider store={getStore()}>
+      <ExpandableImageDialog />
```

Comment:
is there a specific reason this can't just go in `edit-container` alongside `LessonEditor` above?